src/server.py

```diff
@@ -411,6 +428,29 @@ def _ensure_release_countdown(self):
         self._countdown_id = GLib.timeout_add_seconds(
             self._TIMEOUT_S, self.release, priority=GLib.PRIORITY_LOW)
 
+    def _priority_playing(self):
+        for uuid_ in self.players:
+            if self.players[uuid_].priority:
+                return True
+        return False
+
+    def _apply_on_loops(self, uuid, method):
+        if not self.players[uuid].priority:
+            return
+        for _uuid in self.players:
+            if _uuid == uuid:
+                continue
+
+            player = self.players[_uuid]
+            if player.priority:
+                continue
+            if not player.loop:
+                continue
+            if method == 'resume' and not player.paused_by_service:
```

Comment:
You could check that in the `resume` method and do nothing if it's not paused.

Reply:
I had that at some point, same for "paused" and if the sound is already paused. Ill add it back then.